packages/template-express-minimal/package.json

```diff
@@ -1,35 +1,35 @@
 {
-    "name": "template-express-minimal",
-    "version": "2.8.0-dev",
-    "license": "See license in LICENSE",
-    "private": true,
-    "devDependencies": {
-        "pwa-kit-dev": "2.8.0-dev",
-        "pwa-kit-runtime": "2.8.0-dev",
-        "supertest": "^4.0.2"
-    },
-    "scripts": {
-        "test": "pwa-kit-dev test",
-        "format": "pwa-kit-dev format \"**/*.{js,jsx}\"",
-        "lint": "pwa-kit-dev lint \"**/*.{js,jsx}\"",
-        "lint:fix": "npm run lint -- --fix",
-        "start": "pwa-kit-dev start",
-        "build": "pwa-kit-dev build",
-        "push": "npm run build && pwa-kit-dev push",
-        "save-credentials": "pwa-kit-dev save-credentials",
-        "tail-logs": "pwa-kit-dev tail-logs"
-    },
-    "mobify": {
-        "ssrEnabled": true,
-        "ssrParameters": {
-            "ssrFunctionNodeVersion": "16.x"
-        },
-        "ssrOnly": [
-            "ssr.js",
-            "node_modules/**/*.*"
-        ],
-        "ssrShared": [
-            "intentionally-does-not-exist"
-        ]
-    }
+	"name": "template-express-minimal",
```

Comment:
Linting problem? Indentation is 6 spaces

Reply:
Our package.json files don't get linted and I think are inconsistent. They should all be 4 spaces now